Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh

```diff
@@ -72,11 +117,19 @@ for module in "${HYPERV_MODULES[@]}"; do
             LogMsg "Found a kernel matching version for $module module: ${version}"
         else
             LogErr "Error: LIS module $module doesn't match the kernel build version!"
-            SetTestStateAborted
-            exit 0
+            exit_code="1"
         fi
+    else
+         LogErr "Error: LIS module $module is not loaded"
+         exit_code="1"
     fi
 done
 
-SetTestStateCompleted
-exit 0
+if [ "1" -eq "$exit_code" ]; then
+    LogMsg "Exiting with state: TestAborted."
+    SetTestStateAborted
+else
+    LogMsg "Exiting with state: TestCompleted."
+    SetTestStateCompleted
+    exit 0
```

Comment:
Put the exit 0 at the end of the script. 
We need to exit with 0 even if test case fails. Otherwise, LISAv2 will try to run the script again, unless -IgnoreLinuxExitCode switch is given.

Reply:
moved exit 0 to end of the file